Sources/MapboxCoreNavigation/SimulatedLocationManager.swift

```diff
@@ -130,30 +146,62 @@ open class SimulatedLocationManager: NavigationLocationManager {
     
     private var routeProgress: RouteProgress?
     
+    private var _nextDate: Date? = nil
+    private func getNextDate() -> Date {
+        if _nextDate == nil || _nextDate! < Date() {
+            _nextDate = Date()
+        } else {
+            _nextDate?.addTimeInterval(1)
```

Comment:
```suggestion
            _nextDate?.addTimeInterval(TimeInterval(updateIntervalMilliseconds) / 1000.0)
```

Reply:
Next date should monotonically be 1 second greater than previous one. Simulation speed up is achieved by more frequent location updates, but in order for Navigator to handle them correctly, we need to increment timestamp like it is a real update.

Reply:
I know, but I would recomend not hardcodding this